Test_STAR_Edge_Cases.py

```diff
@@ -0,0 +1,52 @@
+import json
+import pandas as pd
+import numpy as np
+from STAR import STAR, TrueTie
+import pytest
+
+
+class TestSTAR:
+    def test_election_with_one_record_only(self):
+        columns = ['Allison', 'Bill', 'Carmen']
+        election = [[5, 2, 1, 4]]
+        ballots = pd.DataFrame(columns=columns, data=election)
+        results = STAR(ballots)
+        assert results['elected'] == ['Allison']
+
+    def test_mismatch_between_number_of_candidates_and_ballots(self):
+        with pytest.raises(ValueError):
+            columns = ['Allison', 'Bill', 'Carmen']
+            election = [[5, 2, 1, 4],
+                        [5, 2, 1, 0],
+           ]
+            ballots = pd.DataFrame(columns=columns, data=election)
+            results = STAR(ballots)
+        # how to trigger error:
+        # expected = raises(ValueError) - mismatch between number of candidates and election ballots:
+        #                                 number of columns in election must be the same as number of candidates;
+        assert(False)
```

Comment:
Does this cause the test to fail or does the error prevent it from reaching this line?